File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.java
class Solution {
    public int[] leftRightDifference(int[] nums) {
        int[] leftsum= new int[nums.length];
        int[] rightsum= new int[nums.length];
        int[] answer= new int[nums.length];
        int count=0;
        for(int i=0; i<nums.length; i++){
            leftsum[i]=count;
            count+=nums[i];
        }
        count=0;
        for(int j=nums.length-1; j>=0; j--){
            rightsum[j]=count;
            count+=nums[j];
        }
        for(int p=0; p<nums.length; p++){
            answer[p]= Math.abs(leftsum[p]-rightsum[p]); 
        }
        return answer;
    }
}

File: 2574-left-and-right-sum-differences/2574-left-and-right-sum-differences.py
class Solution(object):
    def leftRightDifference(self, nums):
        leftsum= []
        rightsum=[]
        rightsum.append('l')
        leftsum.append('j')
        answer=[]
        for j in range(len(nums)):
            if leftsum[0]=='j':
                leftsum[0]=0
                leftsum.append(nums[0])
            else:
                k= leftsum[j]+nums[j]
                leftsum.append(k)
            
        for p in range(len(nums)):
            if rightsum[0]=='l':
                rightsum[0]=0
                rightsum.insert(0, nums[-1])
            else:
                l= rightsum[0]+nums[-1-p]
                rightsum.insert(0, l)
        
        leftsum.pop() 
        rightsum.pop(0)
        for n in range(0, len(nums)):
            tururu= leftsum[n]-rightsum[n]
            answer.append((abs(tururu)))
        
        return answer
    